src/main/java/com/trzewik/spring/domain/game/Game.java

```diff
@@ -244,4 +245,41 @@ public RuntimeException(String msg) {
             super(msg);
         }
     }
+
+    static List<Result> createResults(final Game game) throws Result.Exception {
+        validateGameStatus(game);
+        return generateResults(game);
+    }
+
+    private static void validateGameStatus(final Game game) throws Result.Exception {
+        if (!game.getStatus().isEnded()) {
+            throw new Result.Exception("Results are available only when game finished. Please continue auction.");
+        }
+    }
+
+    private static List<Result> generateResults(Game game) {
+        List<PlayerInGame> players = game.getPlayers();
```

Comment:
inline - you don't need players variable. Just use game.getPlayers().stream()

Reply:
Done